stan/math/rev/scal/fun/rising_factorial.hpp

```diff
@@ -38,33 +33,12 @@ namespace stan {
                - digamma(avi_->val_));
         }
       };
-
-      class rising_factorial_dv_vari : public op_dv_vari {
-      public:
-        rising_factorial_dv_vari(double a, vari* bvi) :
-          op_dv_vari(rising_factorial(a, bvi->val_), a, bvi) {
-        }
-        void chain() {
-          bvi_->adj_ += adj_ * rising_factorial(ad_, bvi_->val_)
-            * digamma(bvi_->val_ + ad_);
-        }
-      };
     }
 
     inline var rising_factorial(const var& a,
-                                double b) {
+                                int b) {
```

Comment:
pack as many arguments as possible onto a line (our style has changed over time).  not a big deal if you can't get to this minor style issue.